pkgs/development/web/mailcatcher/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "mailcatcher-${version}";
+
+  version = (import ./gemset.nix).mailcatcher.version;
+
+  env = bundlerEnv {
+    name = "${name}-gems";
+
+    inherit ruby;
+
+    gemdir = ./.;
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  phases = ["installPhase"];
```

Comment:
I'm unfamiliar with ruby packaging, but is there no fixup phase? By listing the phases you're disabling lots of potentially useful logic.